pkg/resolver/resolver.go

```diff
@@ -0,0 +1,61 @@
+package resolver
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/sonr-io/sonr/pkg/client"
+	"github.com/sonr-io/sonr/pkg/crypto/mpc"
+	"github.com/sonr-io/sonr/pkg/did"
+)
+
+type ResolverImpl struct {
+	wallet *mpc.Wallet
+	client *client.Client
+	did.Resolver
```

Comment:
This is an interface, making a struct inherit from it will suppress any errors for unimplemented methods.